### PR TITLE
updated calendar with paid zoom account URLs

### PR DESCRIPTION
**Pull request by jtfogarty**
updated calendar with paid zoom account URLs

### CODE REVIEW

**Comment by PatrickXYS** [comment]
/lgtm

It would be great if we can merge the PR and update calendar before tomorrow 5:30PM

**Comment by jtfogarty** [comment]
@karlschriek @yuzliu Can you push this through?

**Comment by thesuperzapper** [comment]
Currently this has the wrong links, we need to fix them first

File: calendar/calendar.yaml [hunk 1]
```diff
@@ -17,38 +17,42 @@
 #
 - id: kf001
   name: Kubeflow Community Call
```
Comment (thesuperzapper) [inline calendar/calendar.yaml:19]: Lets call this one: `Kubeflow Community Call (US West/APAC)`

File: calendar/calendar.yaml [hunk 2]
```diff
@@ -17,38 +17,42 @@
...
+      Join from PC, Mac, Linux, iOS or Android: https://us02web.zoom.us/j/83583392870?pwd=Q3lYQlAxbVYrdVZQNlp5cktIY2JmUT09
+      Meeting ID: 835 8339 2870
+      Passcode: 499845
 
       Join from phone in the US: +1 669 900 6833 or +1 646 558 8656
       International numbers available: https://zoom.us/zoomconference?m=Os1EjlUlpb2_XUMaQ6dX1azqMK5CkfWH
   organizer: theadactyl
 
 - id: kf002
   name: Kubeflow Community Call
```
Comment (thesuperzapper) [inline calendar/calendar.yaml:40]: Lets call this one `Kubeflow Community Call (US East/EMEA)`

File: calendar/calendar.yaml [hunk 3]
```diff
@@ -17,38 +17,42 @@
...
+  until: 07/05/2022
+  video: https://us02web.zoom.us/j/83469905816?pwd=aWM5clpLOTNTMU9udFpkZmV0L01VQT09
   attendees:
     - email: kubeflow-discuss@googlegroups.com
   description:
     - |
       Notes & agenda: https://bit.ly/kf-meeting-notes
 
+      Join from PC, Mac, Linux, iOS or Android: https://us02web.zoom.us/j/83469905816?pwd=aWM5clpLOTNTMU9udFpkZmV0L01VQT09
+      Meeting ID: 835 8339 2870
```
Comment (thesuperzapper) [inline calendar/calendar.yaml:53]: This should be: `834 6990 5816`

File: calendar/calendar.yaml [hunk 4]
```diff
@@ -17,38 +17,42 @@
...
+      Passcode: 499845
```
Comment (thesuperzapper) [inline calendar/calendar.yaml:54]: This should be: `683429`

File: calendar/calendar.yaml [hunk 5]
```diff
@@ -17,38 +17,42 @@
...
+
       Join from PC, Mac, Linux, iOS or Android: https://us02web.zoom.us/j/81759654415?pwd=N3NHV3ViN0FxSjkwTWNMUlFvMTQvZz09
```
Comment (thesuperzapper) [inline calendar/calendar.yaml:56]: This line is duplicated, and has the wrong link.

File: calendar/calendar.yaml [hunk 6]
```diff
@@ -17,38 +17,42 @@
...
       Passcode: kubeflow
```
Comment (thesuperzapper) [inline calendar/calendar.yaml:57]: This line needs to be removed

**Comment by thesuperzapper** [comment]
@googlebot I consent.

**Comment by jtfogarty** [comment]
replaced

**Comment by thesuperzapper** [comment]
see #391

**Pull request closed without merging**